utils.ts

```diff
@@ -0,0 +1,44 @@
+import { getMetaTags } from './mod.ts'
+import { MetaTags } from './type.d.ts'
+import { DOMParser, HTMLDocument } from "jsr:@b-fuze/deno-dom";
```

Comment:
We are already using another package to fetch and parse the DOM, can you please use the same one to avoid one more dependency?